<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundPropertyGridWidget</class>
 <widget class="QWidget" name="BackgroundPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="headerPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QLabel" name="backgroundNameLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>0</y>
           <width>280</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>backgroundNameLabel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="expandButton">
         <property name="geometry">
          <rect>
           <x>265</x>
           <y>2</y>
           <width>18</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="detailsPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>340</height>
         </size>
        </property>
        <widget class="QLabel" name="AlignLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>176</y>
           <width>62</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Align:</string>
         </property>
        </widget>
        <widget class="QColorWidget" name="selectColorWidget" native="true">
         <property name="geometry">
          <rect>
           <x>79</x>
           <y>1</y>
           <width>206</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>42</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="drawTypeLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>87</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Draw type:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="openSpriteButton">
         <property name="geometry">
          <rect>
           <x>234</x>
           <y>28</y>
           <width>30</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QComboBox" name="drawTypeComboBox">
         <property name="geometry">
          <rect>
           <x>79</x>
           <y>82</y>
           <width>209</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="tbSpinBox">
         <property name="geometry">
          <rect>
           <x>241</x>
           <y>201</y>
           <width>46</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
        <widget class="QLabel" name="tbStretchLabel">
         <property name="geometry">
          <rect>
           <x>154</x>
           <y>206</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Vert. Stretch:</string>
         </property>
        </widget>
        <widget class="QLabel" name="lrStretchLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>205</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Hor. Stretch:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="lrSpinBox">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>201</y>
           <width>46</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="frameSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>56</y>
           <width>41</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="colorInheritLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>109</y>
           <width>62</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Color 
inherit:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="modificationComboBox">
         <property name="geometry">
          <rect>
           <x>194</x>
           <y>55</y>
           <width>95</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="removeSpriteButton">
         <property name="geometry">
          <rect>
           <x>264</x>
           <y>28</y>
           <width>22</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
        <widget class="QLabel" name="colorLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>4</y>
           <width>62</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
        <widget class="QLabel" name="spriteLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>30</y>
           <width>62</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sprite:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="colorInheritComboBox">
         <property name="geometry">
          <rect>
           <x>79</x>
           <y>111</y>
           <width>209</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="spriteLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>28</y>
           <width>154</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="frameLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>60</y>
           <width>62</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Frame:</string>
         </property>
        </widget>
        <widget class="QLabel" name="modificationLabel">
         <property name="geometry">
          <rect>
           <x>124</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Reflection:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="alignComboBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>209</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="pixelAccuracyLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>141</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Per pixel
accuracy:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="pixelAccuracyComboBox">
         <property name="geometry">
          <rect>
           <x>79</x>
           <y>140</y>
           <width>209</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="UIMarginsPropertyGridWidget" name="marginsWidget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>230</y>
           <width>280</width>
           <height>10</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorWidget</class>
   <extends>QWidget</extends>
   <header>qcolorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>UIMarginsPropertyGridWidget</class>
   <extends>QWidget</extends>
   <header>uimarginspropertygridwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
